src/ducks/categorization/services.js

```diff
@@ -213,24 +216,54 @@ export const localModel = async (classifierOptions, transactions) => {
       'No classifier, impossible to categorize transactions'
     )
     return
+  } else {
+    return classifier
```

Comment:
You should always return `classifier` and check if it is null in the `localModel` function. Here, we can return `undefined` and it seems we don't check it in `localModel` so there's potentially a bug.